.github/workflows/build_quic_interop_container.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   update_quay_container:
+    if: ${{ github.repository == 'openssl/openssl' }}
```

Comment:
this should also work without the braces:
```suggestion
    if: github.repository == 'openssl/openssl'
```

Reply:
Yes, that would work as well, it's a matter of preference (although [not always](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idif))

Do you want me to change it?

Reply:
yes, that syntax looks more simple to me.

Reply:
done